Libraries/WebSocket/WebSocket.js

```diff
@@ -135,6 +135,14 @@ class WebSocket extends EventTarget(...WEBSOCKET_EVENTS) {
     throw new Error('Unsupported data type');
   }
 
+  sendPing(): void {
```

Comment:
It's not part of the JS browser API. Most low level WebSocket implementations expose a `ping` or `sendPing` method. Usually you can add data to the ping that will be sent back as a pong by the server.

Higher level libs like socket.io do not directly expose a method for pinging but instead manages the regular pinging internally: https://github.com/socketio/engine.io-client/blob/2c55b278a491bf45313ecc0825cf800e2f7ff5c1/lib/socket.js#L487